packages/next/bin/next-export.ts

```diff
@@ -53,10 +54,10 @@ if (!existsSync(join(dir, 'pages'))) {
 }
 
 const options = {
-  silent: argv.silent,
-  threads: argv.threads,
-  concurrency: argv.concurrency,
-  outdir: argv.outdir ? resolve(argv.outdir) : resolve(dir, 'out')
+  silent: args['--silent'] || false,
```

Comment:
`--silent` is defined as `String` but defaults to a `Boolean`? Only seems to be referenced as `if (options.silent) {`

Reply:
Great catch